tijanabekic/assignment1/Assignment 1.java

```diff
@@ -0,0 +1,160 @@
+Q1. Given two strings, determine if one is an anagram of the other. Two words are anagrams of
+each other if they are made of the same letters in a different order
+
+boolean isAnagram(String s1, String s2) {
+	if (s1 == null || s1 == null)
+		return false; // alternative: throw new Exception();

```

Comment:
If you throw exceptions, make sure that they contain a reason:
if (s1 == null) {
  throw new IllegalArgumentException("s1 was not set.");
}